upgrade/upgrade-scripts/upgrade

```diff
@@ -113,8 +113,11 @@ function upgrade_in_place() {
 
 	if [[ "$DLPX_UPGRADE_SKIP_VERIFY" != "true" ]]; then
 		"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
-			"$IMAGE_PATH/verify-impl" "$@" ||
-			die "'$IMAGE_PATH/verify-impl' '$*' failed in '$CONTAINER'"
+			"$IMAGE_PATH/verify-stack" "$@" ||
```

Comment:
I think this should be `verify-jar` rather than `verify-stack`? same for the error message one line below.